src/extension.ts

```diff
@@ -70,24 +76,30 @@ function stylefmtProcess(document: vscode.TextDocument, range: vscode.Range, con
  * Resolve Stylefmt config
  */
 function useStylefmt(document: vscode.TextDocument, range: vscode.Range): Promise<IResult> {
-	const settings: IStylefmtOptions = vscode.workspace.getConfiguration().get('stylefmt');
+	const settingsFmt: IStylefmtOptions = vscode.workspace.getConfiguration().get('stylefmt');
+	const settingsLint: IStylelintOptions = vscode.workspace.getConfiguration().get('stylelint');
 	const configResolver = new ConfigResolver(vscode.workspace.rootPath);
-	const options = {
+	const resolveOptions = {
 		packageProp: 'stylelint',
 		configFiles: [
 			'.stylelintrc',
 			'stylelint.config.js'
 		],
-		editorSettings: settings.config
+		editorSettings: settingsFmt.config
 	};
 
-	return configResolver.scan(document.uri.fsPath, options).then((resolved) => {
+	let configOverrides = null;
+	if (settingsFmt.useStylelintConfigOverrides) {
+		configOverrides = settingsLint.configOverrides;
+	}
+
+	return configResolver.scan(document.uri.fsPath, resolveOptions).then((resolved) => {
 		return stylefmtProcess(document, range, {
-			configBasedir: settings.configBasedir || vscode.workspace.rootPath,
-			config: resolved.json
+			configBasedir: settingsFmt.configBasedir || vscode.workspace.rootPath,
+			config: extend(true, {}, resolved.json, { rules: configOverrides || {} })
```

Comment:
Issue with Object.assign is that it doesn't deep copy, so nested properties will be lost rather than overwritten. We could write our own extender, but I didn't see the benefit in that.